### ISSUE TITLE
Please merge upstream changes

### ISSUE THREAD

**Issue by nin0-dev** [polymorphicshade/NewPipe#155]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Basically there is a nasty bug that when going full screen, NewPipe may switch to landscape mode (even out of full screen) and only way to go back to normal is clearing the app.

It got fixed in new NewPipe commits, so it would be a good idea for @polymorphicshade to merge upstream changes and release a new version here.

**Comment by polymorphicshade** [comment]
In the interest of stable code, the only time I will merge anything from master is when a new release is sent out. You will have to wait patiently for a new release of NewPipe before I merge in anything new.

**Comment by Atemu** [comment]
Backporting bug fixes to stable via cherry-pick might be a good idea on some cases though as upstream seems to be very sluggish about that. 

It won't cause merge conflicts later either because it's 1:1 the same change and git is smart enough to recognise that. No real maintenance burden here.

I often do that in my personal trees to check out new features or get bug fixes early independently of upstream's release schedules.
